tests/unit/activity.test.ts

```diff
@@ -14,7 +15,7 @@ describe("Activity", () => {
 
   describe("Creating", () => {
     it("should create activity", async () => {
-      const activity = await Activity.create("test_agreement_id", yagnaApi);
+      const activity = await Activity.create(agreement, yagnaApi);
```

Comment:
The fact that `yagnaApi` has to be passed here all the time is a bit disturbing.